### DISCUSSION TITLE
readValues and correspondence

### DISCUSSION THREAD

**Discussion by winhkey** [eclipse-milo/milo#1091] [General]
List<NodeId> list = .....;
CompletableFuture<List<DataValue>> readValues = connect.readValues(0.0, Both, list);

Is it possible the each count of NodeId list and DataValue list are different, and how can I correspondence these NodeId and DataValue one to one？

**Comment by kevinherron** [comment]
The resulting list should be the same size and the indices should correspond to those of the request.

**Reply by winhkey** [reply]
ok thx
